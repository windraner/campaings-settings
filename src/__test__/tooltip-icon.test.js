import React from 'react';
import TooltipIcon from '../components/main-content-section/tab-block/common/tooltip-icon/tooltip-icon';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

describe('<TooltipIcon />', () => {
  it('Check for className and text', () => {
    const wrapper = shallow(<TooltipIcon text="test" />);
    expect(wrapper.find('.campaigns-settings__tooltip-icon').length).toBe(1);
    expect(wrapper.find('.campaigns-settings__tooltip-icon').text()).toBe('test');
  });
  it('matches the snapshot hide', () => {
    const tree = shallow(<TooltipIcon text="test" />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
