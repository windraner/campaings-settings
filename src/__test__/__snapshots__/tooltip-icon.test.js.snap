// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TooltipIcon /> matches the snapshot hide 1`] = `
<span
  className="campaigns-settings__tooltip-icon my-auto d-flex justify-content-center"
>
  <span
    className="campaigns-settings__tooltip position-absolute"
  >
    test
  </span>
</span>
`;
